src/store/index.ts

```diff
@@ -1,8 +1,15 @@
 import { createStore } from "vuex";
+import { QuestionBucketingMap } from "../types"
 
 export default createStore({
   state: {
     isSpinnerShown: false,
+    questionsBucketingMap: {} as QuestionBucketingMap,
+    bucketSize: 10,
+  },
+  getters: {
+    questionBucketingMap: (state) => state.questionsBucketingMap,
```

Comment:
why do you need a getter?
instead of accessing it as store.getters.questionBucketingMap, you can simply access it as store.state['questionsBucketingMap']. Getters are used when we are doing something on top of some state value or using multiple getters.